docker-compose.yml

```diff
@@ -1,11 +1,24 @@
-version: '3.8'
+version: "3.8"
 
 services:
+  beacons-service:
+    build:
+      context: ./
+    container_name: app
+    ports:
+      - 8080:8080
+    depends_on:
+      - postgres
+    environment:
+      SPRING_DATASOURCE_URL: jdbc:postgresql://postgres:5432/beacons
+      SPRING_DATASOURCE_USERNAME: beacons-service
+      SPRING_DATASOURCE_PASSWORD: password
```

Comment:
You can leave these out as they are set to default to these values within the application so everything should work as is.  We have to update the datasource url as the hostname is not `localhost` but rather the hostname within the Docker network which will be `postgres` which is why we had to update that variable :)